gems/smithy/lib/smithy/anvil/client/templates/plugins/endpoint.erb

```diff
@@ -33,7 +34,7 @@ module <%= namespace %>
       # @api private
       class Handler < Smithy::Client::Handler
         def call(context)
-          params = EndpointParameters.create(context.config)
+          params = EndpointParameters.create(context.config, context.operation_name, context.params)
```

Comment:
This is a private method right? Why not just take "context" here and have the method unpack these?

Reply:
Its an api private method.  I guess I can just take context, but I had wanted to avoid coupling the endpoint parameters to context directly.

Reply:
I've updated it to context.